### PR TITLE
Use translation keys for Network configuration pages

### LINKED ISSUE #13464: Network configuration page doesn't use translation keys fully
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

Network configuration page doesn't use translation keys in these parts:
* image
* image
* image
* image

Excerpts without translation keys:
* https://github.com/home-assistant/frontend/blob/be169f9c83222962b37f6e3d808d2e0feab80d39/src/panels/config/network/supervisor-hostname.ts#L58-L62

* https://github.com/home-assistant/frontend/blob/be169f9c83222962b37f6e3d808d2e0feab80d39/src/panels/config/network/supervisor-network.ts#L247-L248

* https://github.com/home-assistant/frontend/blob/be169f9c83222962b37f6e3d808d2e0feab80d39/src/panels/config/network/dialog-ip-detail.ts#L44

* https://github.com/home-assistant/frontend/blob/be169f9c83222962b37f6e3d808d2e0feab80d39/src/panels/config/network/dialog-ip-detail.ts#L49-L59

* https://github.com/home-assistant/frontend/blob/be169f9c83222962b37f6e3d808d2e0feab80d39/src/panels/config/network/dialog-ip-detail.ts#L66-L79

* https://github.com/home-assistant/frontend/blob/be169f9c83222962b37f6e3d808d2e0feab80d39/src/panels/config/network/ha-config-network.ts#L43

* https://github.com/home-assistant/frontend/blob/be169f9c83222962b37f6e3d808d2e0feab80d39/src/panels/config/network/ha-config-network.ts#L52-L56

### Describe the behavior you expected

It was expected that the frontend would use translation keys and they would be available in Lokalise.

### Steps to reproduce the issue

1. Change the system language to one other than English.
2. Access the analytics settings page: Settings > System > Network

### What version of Home Assistant Core has the issue?

2022.8.6

### What was the last working version of Home Assistant Core?

_No response_

### In which browser are you experiencing the issue with?

104.0.5112.102

### Which operating system are you using to run this browser?

Windows 10

### State of relevant entities

_No response_

### Problem-relevant frontend configuration

_No response_

### Javascript errors shown in your browser console/inspector

_No response_

### Additional information

_No response_

### LINKED ISSUE #17212: Missing translations in IP details dialog
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

Extracted from: https://github.com/home-assistant/core/issues/94345

image

Ref: https://github.com/home-assistant/frontend/blob/3191801fa7c4bb5487dd08346f94bb304a9b669b/src/panels/config/network/dialog-ip-detail.ts#L44-L75

### Describe the behavior you expected

Those to be translateable.

### Steps to reproduce the issue

1. Open the network ip details dialogs

### What version of Home Assistant Core has the issue?

core-2023.8.0dev0

### What was the last working version of Home Assistant Core?

_No response_

### In which browser are you experiencing the issue with?

_No response_

### Which operating system are you using to run this browser?

_No response_

### State of relevant entities

_No response_

### Problem-relevant frontend configuration

_No response_

### Javascript errors shown in your browser console/inspector

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by silamon**
## Proposed change
Update the labels to make them translatable. <s>This change isn't fully tested at this time. </s>

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #13464 fixes #17212
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by steverep** [comment]
Please use imperative statements for your PR titles (not the issue title) as this will become the squashed commit message everyone will see in the release notes.  It also helps reviewers get a good idea of what your PR is trying to achieve right off the bat.

https://developers.home-assistant.io/docs/development_submitting/

File: src/panels/config/network/dialog-ip-detail.ts [hunk 1]
```diff
@@ -41,20 +41,49 @@ class DialogIPDetail extends LitElement {
...
+                <h3>
+                  ${this.hass.localize("ui.dialogs.dialog-ip-detail.ipv4")}
+                </h3>
                 ${ipv4.address
-                  ? html`<div>IP Address: ${ipv4.address?.join(", ")}</div>`
+                  ? html`<div>
+                      ${this.hass.localize(
+                        "ui.dialogs.dialog-ip-detail.ip_address"
+                      )}:
+                      ${ipv4.address?.join(", ")}
```
Comment (bramkragten) [inline src/panels/config/network/dialog-ip-detail.ts:60]: Pass the IP addresses as a param to the translation string:
```suggestion
                      ${this.hass.localize(
                        "ui.dialogs.dialog-ip-detail.ip_address", {address: ipv4.address?.join(", ")}
                      )}
```

File: src/panels/config/network/dialog-ip-detail.ts [hunk 2]
```diff
@@ -63,15 +92,41 @@ class DialogIPDetail extends LitElement {
...
+                <h3>
+                  ${this.hass.localize("ui.dialogs.dialog-ip-detail.ipv6")}
+                </h3>
                 ${ipv6.address
-                  ? html`<div>IP Address: ${ipv6.address?.join(", ")}</div>`
+                  ? html`<div>
+                      ${this.hass.localize(
+                        "ui.dialogs.dialog-ip-detail.ip_address"
+                      )}:
+                      ${ipv6.address?.join(", ")}
```
Comment (bramkragten) [inline src/panels/config/network/dialog-ip-detail.ts:103]: Same

File: src/panels/config/network/dialog-ip-detail.ts [hunk 3]
```diff
@@ -63,15 +92,41 @@ class DialogIPDetail extends LitElement {
...
+                    </div>`
+                  : ""}
+                ${ipv6.gateway
+                  ? html`<div>
+                      ${this.hass.localize(
+                        "ui.dialogs.dialog-ip-detail.gateway"
+                      )}:
+                      ${ipv6.gateway}
```
Comment (bramkragten) [inline src/panels/config/network/dialog-ip-detail.ts:111]: Same

File: src/panels/config/network/dialog-ip-detail.ts [hunk 4]
```diff
@@ -63,15 +92,41 @@ class DialogIPDetail extends LitElement {
...
-                ${ipv6.gateway ? html`<div>Gateway: ${ipv6.gateway}</div>` : ""}
-                ${ipv6.method ? html`<div>Method: ${ipv6.method}</div>` : ""}
                 ${ipv6.nameservers?.length
                   ? html`
-                      <div>Name Servers: ${ipv6.nameservers?.join(", ")}</div>
+                      <div>
+                        ${this.hass.localize(
+                          "ui.dialogs.dialog-ip-detail.name_servers"
+                        )}:
+                        ${ipv6.nameservers?.join(", ")}
```
Comment (bramkragten) [inline src/panels/config/network/dialog-ip-detail.ts:128]: Same

**Review by piitaya — APPROVED** [review]

**Comment by piitaya** [comment]
Thanks!

**Pull request merged**